src/app/services/catalogue/catalogue.service.ts

```diff
@@ -26,16 +32,51 @@ export class CatalogueService {
   }
 
   private getLocalItems(tab: Tab): Observable<Repository[]> {
-    const { org = '', topic = '' } = tab;
+    const { org = 'azerothcore', topic = '' } = tab;
```

Comment:
not sure about default value, I would keep it empty to keep the project agnostic

we can even merge this PR but we have to rework this project to make it more agnostic in general